browser/extensions/brave_extension_provider.cc

```diff
@@ -29,8 +29,8 @@ bool IsBlacklisted(const extensions::Extension* extension) {
        "mlklomjnahgiddgfdgjhibinlfibfffc",
      });
 
-  if (std::find(blacklisted_extensions.begin(), blacklisted_extensions.end(),
-                extension->id()) != blacklisted_extensions.end())
+  if (base::ranges::find(blacklisted_extensions, extension->id()) !=
+      blacklisted_extensions.end())
```

Comment:
base::Contains here also
and everywhere below where we compare with .end() iterator

Reply:
Fixed.